### ISSUE TITLE
need to unblock for Youtube on Safari

### ISSUE THREAD

**Issue by Tonyy881** [AdguardTeam/AdGuardForSafari#922]
### Please answer the following questions for yourself before submitting an issue.

- [X] I am running the latest version
- [X] I made sure all content blockers are enabled in Safari settings
- [X] I checked the documentation and found no answer
- [X] I checked to make sure that this issue has not already been filed

### AdGuard version

1.11.14

### Environment

```markdown
- OS:Sonoma 14
- Safari version:17.0
```

### What filters do you have enabled?

AdBlockID

### Issue Details

Steps to reproduce:
1.
2.
3.

### Actual Behavior

Video will not play, because violate YouTube's terms of Service.

### Screenshots

<details><summary>Screenshot 1:</summary>

<img width="993" alt="Screenshot 2023-10-23 at 3 54 33 PM" src="https://github.com/AdguardTeam/AdGuardForSafari/assets/148810158/2bb4b15e-2e61-4617-be90-bbb9dfe8042a">

### Additional Information

I need a detail of instructions for how to solve the issue.

**Comment by polarstoat** [comment]
This appears to be a duplicate of #913
